integration/kubernetes/helm-chart/alluxio/templates/master/statefulset.yaml

```diff
@@ -194,8 +194,23 @@ spec:
             claimName: "{{ .name }}"
           {{- end }}
         {{- end }}
+  {{- if $isRoot }}
+  volumes:
+    {{- if $needJournalVolume }}
+    - name: alluxio-journal
+      hostPath:
```

Comment:
I don't think we still want to consider `emptyDir`, do we?

Reply:
no